bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_LO_MLM/DY_HT-100to200_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-no_masses
```

Comment:
Yes.  Makes no difference for dy/z (but also no disadvantage, so might as well stay consistent)

For W it actually has some effect on the flavour combinations, and speed-wise we can afford it, so might as well.  (But here was tested only with the decay chain syntax + large bwcutoff, which helps keep the speed acceptable)